zokrates_core/src/imports.rs

```diff
@@ -149,6 +149,18 @@ impl Importer {
             // handle the case of special bellman and packing imports
             if import.source.starts_with("EMBED") {
                 match import.source.to_str().unwrap() {
+                    #[cfg(feature = "bellman")]
```

Comment:
Am I following correctly that we can't refuse this embed here as we did not introduce T yet?